app/concepts/cinema_halls/representers/all.rb

```diff
@@ -0,0 +1,23 @@
+module CinemaHalls
+  module Representers
+    class All
+      attr_reader :cinema_halls
+
+      def initialize(cinema_halls)
+        @cinema_halls = cinema_halls
+      end
+
+      def basic
+        cinema_halls.map do |cinema_hall|
+          Single.new(cinema_hall).basic
+        end
+      end
+
+      def extended
+        cinema_halls.map do |cinema_hall|
+          Single.new(cinema_hall).extended
```

Comment:
You don't have `extended` method in `CinemaHalls::Representers::Single` class.

Reply:
Good point, thanks! I have an idea to return basic (just cinema_hall object) and extended (cinema_hall with screenings) on request in endpoint:
```
  def index
    cinema_halls = CinemaHalls::Repository.new.find_all

    if permit_params[:extended]
      render json: CinemaHalls::Representers::All.new(cinema_halls).extended
    else
      render json: CinemaHalls::Representers::All.new(cinema_halls).basic
    end
  end
```

but I'm not sure if it would be good to create something like this

Reply:
That should be fine 👍